docs/usage/build_fms_model.rst

```diff
@@ -119,15 +110,9 @@ In order for the container to build successfully, the parallel checkout feature
   # Create checkout script
   fre make checkout-script -y null_model.yaml -p hpcme.2023 -t prod
 
-  # Create and run checkout script
-  fre make checkout-script -y null_model.yaml -p hpcme.2023 -t prod --execute
-
   # Create Makefile
   fre make makefile -y null_model.yaml -p hpcme.2023 -t prod
 
-  # Create Dockerfile
-  fre make dockerfile -y null_model.yaml -p hpcme.2023 -t prod
-
   # Create and run the Dockerfile
```

Comment:
I think we could be more detailed: "Create a dockerfile and a container build script and execute the container build script [filename]" I can't easily ascertain what the filename of the container build script is - that would be helpful to use the filename where I have [filename]